index.html

```diff
@@ -4,13 +4,174 @@
-    <title>Document</title>
+    <title>Capstone-1</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="desktop.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+    <!-- font awesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
 </head>
 <body>
+    <!-- -------------------- NAVBAR -------------------- -->
     <header>
-        <nav>
-            <h1>hello</h1>
+        <div class="top-header">
+            <img src="./images/icons/facebook-logo-24.png" alt="facebook">
+            <img src="./images/icons/twitter-logo-24.png" alt="twiter-icon">
+            <ul>
+                <li>English</li>
+                <li>My page</li>
+                <li>Logout</li>
+            </ul>
+        </div>
+        <nav class="nav-cover">
+          <div class="menu-icon"><img src="./images/icons/menu-regular-24.png" alt=""></div>
+          <div class="desktop-nav">
+              <h2 class="logo">The <span>Coder</span></h2>
+                <a href="./about.html">About</a>
+                <a href="#main-program">Program</a>
+                <a href="#">Join</a>
+                <a href="#partners">Sponsor</a>
+                <a href="#">News</a>
+                <a href="#" class="get-started">Get started</a>
+            </div>
+            <div class="mobile-menu hide">
+                <ul>
+                    <li><a href="./about.html" class="menu-link">About</a></li>
+                    <li><a href="#main-program" class="menu-link">Program</a></li>
+                    <li><a href="#featured-speaker" class="menu-link">Speakers</a></li>
+                    <li><a href="#partners" class="menu-link">Partners</a></li>
+                </ul>
+                <div class="cancle-mobile hide">
+                    <img src="./images/icons/Icon.svg" alt="cancle-icon">
+                </div>
+            </div>
         </nav>
     </header>
+    <!-- -------------------- Hero -------------------- -->
+    <section id="hero">
+        <div class="hero-container">
+            <h2>The coder</h2>
+            <h1 class="details">
+                CREATIVE COMMON <br> GLOBAL SUMMIT 2023
+            </h1>
+            <p class="info">A joyful celebration beliving in the value of openness and sharring a positive
+                change with people from over 80 countries in the world.
+            </p>
+            <h3>2023.10.15(THU) ~ 17(SAT)</h3>
+            <h4 class="loction">@ National Confrence center of Africa, Creative center.</h4>
```

Comment:
- [ ] Good job adding this text as requested by the previous reviewer. Kindly make the text have the same font-weight as the given design so as to meet the requirements.

Your design
![image](https://github.com/Shabiblac/CAPSTONE-1/assets/81932478/67cf84b7-eb10-4d54-a9a4-c6b20217da7e)

Given design
![image](https://github.com/Shabiblac/CAPSTONE-1/assets/81932478/cf3fc76d-e67a-459e-86f8-cd3c91bed362)

## Note that the previous reviewer urged you to add this paragraph with its style
![image](https://github.com/Shabiblac/CAPSTONE-1/assets/81932478/5eacab49-e781-4ddd-9be3-84c503d244d4)